ovh/resource_cloud_project_volume_gen.go

```diff
@@ -0,0 +1,795 @@
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+	ovhtypes "github.com/ovh/terraform-provider-ovh/ovh/types"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+func CloudProjectVolumeResourceSchema(ctx context.Context) schema.Schema {
+	attrs := map[string]schema.Attribute{
+		"action": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Computed:            true,
+			Description:         "The action of the operation",
+			MarkdownDescription: "The action of the operation",
+		},
+		"completed_at": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Computed:            true,
+			Description:         "The completed date of the operation",
+			MarkdownDescription: "The completed date of the operation",
+		},
+		"created_at": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Computed:            true,
+			Description:         "The creation date of the operation",
+			MarkdownDescription: "The creation date of the operation",
+		},
+		"description": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Optional:            true,
+			Computed:            true,
+			Description:         "Volume description",
+			MarkdownDescription: "Volume description",
+		},
+		"id": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Computed:            true,
+			Description:         "Operation ID",
+			MarkdownDescription: "Operation ID",
+		},
+		"image_id": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Optional:            true,
+			Computed:            true,
+			Description:         "Image ID",
+			MarkdownDescription: "Image ID",
+		},
+		"instance_id": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Optional:            true,
+			Computed:            true,
+			Description:         "Instance ID",
+			MarkdownDescription: "Instance ID",
+		},
+		"name": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Optional:            true,
+			Computed:            true,
+			Description:         "Volume name",
+			MarkdownDescription: "Volume name",
+		},
+		"progress": schema.Int64Attribute{
+			CustomType:          ovhtypes.TfInt64Type{},
+			Computed:            true,
+			Description:         "Volume status",
+			MarkdownDescription: "Volume status",
+		},
+		"region_name": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Required:            true,
+			Description:         "Region name",
+			MarkdownDescription: "Region name",
+		},
```

Comment:
```suggestion
		"region_name": schema.StringAttribute{
			CustomType:          ovhtypes.TfStringType{},
			Required:            true,
			Description:         "Region name",
			MarkdownDescription: "Region name",
			PlanModifiers: []planmodifier.String{
				stringplanmodifier.RequiresReplace(),
			},
		},
```

and same for other fields that should trigger a replace (at least `service_name`, `size` and `type`) if the doc is correct